parlai/mturk/core/dev/agents.py

```diff
@@ -90,76 +80,13 @@ def is_final(self):
             self.status == self.STATUS_DISCONNECT
             or self.status == self.STATUS_DONE
             or self.status == self.STATUS_PARTNER_DISCONNECT
-            or self.status == self.STATUS_PARTNER_DISCONNECT_EARLY
+            or
+            # TODO REMOVE THIS TYPE
+            self.status == self.STATUS_PARTNER_DISCONNECT_EARLY
             or self.status == self.STATUS_RETURNED
             or self.status == self.STATUS_EXPIRED
         )
 
-    def get_inactive_command_text(self):
```

Comment:
All of these have been moved to being managed on the server-side in `status_messages.js`. An agent is really only responsible for its own state, not necessarily what we're going to be displaying on the frontend in particular states.